.github/container/test-pax.sh

```diff
@@ -345,10 +373,17 @@ export XLA_PYTHON_CLIENT_MEM_FRACTION=${XLA_PYTHON_CLIENT_MEM_FRACTION:-0.65}
 export ENABLE_TE=$ENABLE_TE
 export NVTE_FUSED_ATTN=$NVTE_FUSED_ATTN
 
-CONFIG=ci_configs.Synthetic126MCI
-if [[ ${RUN_5B} -ne 0 ]]; then
+if [[ ${MODEL_TYPE} == "126M" ]]; then
+  CONFIG=ci_configs.Synthetic126MCI
+elif [[ ${MODEL_TYPE} == "5B" ]]; then
   CONFIG=paxml.contrib.gpu.scripts_gpu.configs.Synthetic5B
   ADDITIONAL_ARGS="--fdl.DCN_MESH_SHAPE=[1,${NODES},1] --fdl.ICI_MESH_SHAPE=[${DP},${FSDP},${TP}] ${ADDITIONAL_ARGS} --fdl.PERCORE_BATCH_SIZE=${BATCH_PER_GPU}"
+elif [[ ${MODEL_TYPE} == "LLaMA70B" ]]; then
```

Comment:
nit: can the name here be something obvious to indicate it's a proxy model, like `LLaMA70BSmall` or `LLaMA70BProxy`?